alfresco-transformer-base/src/main/java/org/alfresco/transformer/config/WebApplicationConfig.java

```diff
@@ -78,4 +79,10 @@ public TransformServiceRegistry transformRegistry()
     {
         return new TransformRegistryImpl();
     }
```

Comment:
We need to avoid adding debug to the clientData from both t-router and t-engines. This flag is used.